include/types/global.hpp

```diff
@@ -4,10 +4,15 @@
 #include "util/print.hpp"
 
 namespace tiledcuda::cell {
-
 namespace tl = tile_layout;
 
-template <typename Element_, typename Layout_>
+// FIXME(haruhi): The last boolean template parameter kSwizzled_ is not
+// meaningful for GlobalTile. This issue arises from the current implementation
+// of TileIterator, which always creates a SharedTile containing a kSwizzled_
+// flag. This is a temporary workaround to make TileIterator functional for both
+// GlobalTile and SharedTile. This should be addressed and properly fixed in the
+// future.
+template <typename Element_, typename Layout_, const bool kSwizzled_ = false>
```

Comment:
I'm curious why the `kSwizzled` template parameter needs to be added to `GlobalTile`?

Reply:
This is a buggy workaround, and I add a fixme comment to warn the potential problem.

Reply:
> I'm curious why the `kSwizzled` template parameter needs to be added to `GlobalTile`?

The problem is TileIterator should be work with both GlobalTile and SharedTile:

https://github.com/TiledTensor/TiledCUDA/blob/3b661af42217897ede24cfcb7760cf0da2998c50/include/types/tile_iterator.hpp#L73

and we have this known issue https://github.com/TiledTensor/TiledCUDA/issues/100.

`TileIterator` is used to create a new `Global/SharedTile` type for sub-tiles that records a modified shape and strides for these sub-tiles. This workarounds in fact, leverage `SharedTile` to carry these shape information which is a little bit buggy.

I will fix it throughly in next modifications.